TTHAnalysis/macros/prepareEventVariablesFriendTree.py

```diff
@@ -504,11 +509,11 @@ def analyze(self,event):
                 cmd = "echo \"{base} -d {data} {post}\" | {super} {writelog}".format(super=super, writelog=writelog, base=basecmd, data=name, chunk=chunk, post=friendPost)
             elif options.env == "oviedo":
                 cmd = "{super} {base} -d {data} {post} {writelog}".format(super=super, writelog=writelog, base=basecmd, data=name, chunk=chunk, post=friendPost)
-            if options.queue == "cp3" and options.env == "uclouvain":
+            if options.queue == "cp3" and ("uclouvain" in options.env):
                 full_subfile = "{subfile}{data}_{chunk}.sh".format(subfile=options.subfile, data=name, chunk=chunk)
-                subfile = open(full_subfile, "w")
+                subfile = open(full_subfile, "w")#--ntasks=2
                 subfile.write("""#! /bin/bash
-#SBATCH --ntasks=8
+#SBATCH 
```

Comment:
This breaks our batch, maybe we can put a conditional

Reply:
@sscruz

Reply:
Unfortunately in uclouvain yes, the way it's done in lxbatch_runner doesn't work :(

Reply:
then i would rather change lxbatch_runner instead and avoid producing n files, but ok, i'm not gonna use it myself :)